src/edu/cscc/javaadventure/MovableObjectImpl.java

```diff
@@ -0,0 +1,16 @@
+package edu.cscc.javaadventure;
+
+public class MovableObjectImpl extends MovableObject {
```

Comment:
MovableObjectImpl is not needed, everything can be tested through the objects that extend movableObject